css/style.css

```diff
@@ -472,16 +536,126 @@ textarea[type="text"]:hover {
   justify-content: center;
 }
 
-.social-icons {
-  display: flex;
-  margin-bottom: 20px;
+.form {
+  /* Center the form on the page */
+  margin: 0 auto;
+  width: 400px;
+
+  /* Form outline */
+  padding: 1em;
+  border: 1px solid #ccc;
+  border-radius: 1em;
 }
 
-.social-icons li {
-  margin-right: 10px;
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+label {
+  /* Uniform size & alignment */
+  display: inline-block;
+  width: 90px;
+  text-align: right;
+  margin-right: 20px;
+}
+
+input,
+textarea {
+  font: 1em sans-serif;
+  width: 300px;
+  box-sizing: border-box;
+  border: 1px solid #999;
+  margin-top: 10px;
+}
+
+input:focus,
+textarea:focus {
+  border-color: #000;
+}
+
+.textarea {
+  vertical-align: top;
+  height: 5em;
+  margin-right: 20px;
+}
+
+.button {
+  margin-left: 760px;
+  margin-top: 35px;
+}
+
+.social-icons {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  margin: 0 20 0 0;
+  padding: 5px;
+  justify-content: center;
+  color: #f5f5ff;
+  column-gap: 0.5rem;
+  margin-top: -20px;
+  margin-right: 60px;
 }
 
 .email-text {
-  margin-bottom: 30px;
+  width: 220px;
+  height: 29px;
+  font-family: 'Roboto', serif;
+  font-style: normal;
+  font-weight: 400;
   font-size: 20px;
+  line-height: 28px;
+  padding-right: 500px;
+  margin-left: 850px;
+  margin-top: -35px;
+
+  /* or 140% */
+  text-align: center;
+
+  /* Primary/B50 */
+  color: #ebebff;
 }
+
+#jeremy,
+#lastname {
+  width: 148px;
+}
+
+.works {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.work-container {
+  display: flex;
+  height: 40vh;
+  margin-bottom: 4rem;
+}
+
+.work-container:nth-of-type(even) {
+  flex-direction: row-reverse;
+}
+
+.project-img,
+.project-details {
+  height: 20rem;
+  margin-right: 20px;
+}
```

Comment:
Please update the margin and the text alignment of the elements inside this box 😄. This will make your project more similar to the one displayed on Figma.
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199632928-b1a67197-60a7-48c7-8b45-9694f6a43647.png)  | ![image](https://user-images.githubusercontent.com/42799579/199632918-89f2954c-5152-4c18-b779-8e4ce2e3b903.png)  |

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.